src/v/config/configuration.cc

```diff
@@ -547,6 +547,47 @@ configuration::configuration()
       "Interval between iterations of controller backend housekeeping loop",
       required::no,
       1s)
+  , compaction_ctrl_update_interval_ms(
+      *this, "compaction_ctrl_update_interval_ms", "", required::no, 30s)
+  , compaction_ctrl_p_coeff(
+      *this,
+      "compaction_ctrl_p_coeff",
+      "proportional coefficient for compaction PID controller. This has to be "
+      "negative since compaction backlog should decrease when number of "
+      "compaction shares increases",
+      required::no,
+      -12.5)
```

Comment:
How did you choose the PID coefficients?

Reply:
It is calculated. More details in `application.cc`